apps/web/src/services/auth/password/PasswordPage.vue

```diff
@@ -0,0 +1,289 @@
+<template>
+    <centered-page-layout>
+        <p-data-loader
+            class="password-page"
+            :loading="state.loading"
+        >
+            <div class="contents-wrapper">
+                <div class="headline-wrapper">
+                    <h1 class="title">
+                        {{ state.pageTitle }}
+                    </h1>
+                    <div class="help-text-wrapper">
+                        <p v-if="props.status === AUTH_ROUTE.PASSWORD.STATUS.RESET._NAME"
+                           class="help-text"
+                        >
+                            {{ $t('AUTH.PASSWORD.RESET.HELP_TEXT') }}
+                            <span class="emphasis">
+                                {{ state.userInfo.userId }}
+                            </span>
+                        </p>
+                        <p v-else
+                           class="help-text"
+                        >
+                            {{ $t('AUTH.PASSWORD.FIND.HELP_TEXT') }}
+                        </p>
+                    </div>
+                </div>
+                <password-form
+                    ref="passwordFormEl"
+                    v-model="formState"
+                    :status="props.status"
+                    @change-input="handleChangeInput"
+                    @click-input="handleClickButton"
+                />
+                <div class="button-wrapper">
+                    <p-button
+                        v-if="props.status === AUTH_ROUTE.PASSWORD.STATUS.RESET._NAME"
```

Comment:
suggestion: It seems necessary to manage the status type, not the route name. Like this?!

```typescript
type Status = 'reset' | 'find'
```